packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/mapper/ServiceClientMapper.java

```diff
@@ -209,8 +209,10 @@ protected List<ServiceClientProperty> processClientProperties(Client client, Str
                 continue;
             }
 
-            String serviceClientPropertyDescription =
+            String description =
                     p.getDescription() != null ? p.getDescription() : p.getLanguage().getJava().getDescription();
+            String summary = p.getSummary();
+            description = SchemaUtil.mergeSummaryWithDescription(summary, description);
```

Comment:
merge summary and description

This could affect code from Swagger code path.

Reply:
just curious, this does not have any diff on our test code?

Reply:
No.

My guess is that currently the only client parameter be `endpoint` and `api-version`. Maybe + `subscriptionId`. Them either be standard one covered by trait, or not having summary.